packages/components/src/components/ContentCard/ContentCard.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import type { Meta, StoryFn } from "@storybook/react";
+import React from "react";
+import { ContentCard } from "./ContentCard";
+
+export default {
+  component: ContentCard,
+  title: "Components/Content Card",
+} as Meta<typeof ContentCard>;
+
+const Template: StoryFn<typeof ContentCard> = (args) => (
+  <ContentCard {...args} />
+);
+
+const linkHref = "https://localyze.com";
+
+const defaultProps = {
+  title: "The card title",
+  text: "Lorem ipsum dolor sit, consectetur. Aliquam ac a. Ligula at et, sodales vel purus.",
+  date: "Mon, Sept 5 2022",
+  tag: "🏠 Permanent Housing",
+  buttonText: "Go to Page",
+  imageSrc: "/images/beach-seal-rocks.jpg",
+};
+
+export const Default = Template.bind({});
```

Comment:
What do you think about adding one more example rendering more than 1 card? Does it make sense? 🤔